scripts/populate-popular-and-new-and-noteworthy-projects.py

```diff
@@ -0,0 +1,140 @@
+"""
+This will update node links on POPULAR_LINKS_NODE and NEW_AND_NOTEWORTHY_LINKS_NODE.
+"""
+import sys
+import logging
+import datetime
+import dateutil
+import operator
+import requests
+from modularodm import Q
+from website.app import init_app
+from website import models
+from framework.auth.core import Auth
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+from website.settings.defaults import POPULAR_LINKS_NODE, NEW_AND_NOTEWORTHY_LINKS_NODE
```

Comment:
Nope, I was in thinking of the wrong context, the web app. Here in the script context we do indeed take `defaults` values without looking at `local`. We should import from `website.settings` to allow for setting these values in `website.settings.local` instead of overwriting `website.settings.defaults`.

```
(Pdb) l
 14     from scripts import utils as script_utils
 15     from framework.transactions.context import TokuTransaction
 16     from website.settings.defaults import POPULAR_LINKS_NODE, NEW_AND_NOTEWORTHY_LINKS_NODE
 17     import pdb; pdb.set_trace()
 18     
 19  -> logger = logging.getLogger(__name__)
 20     
 21     # TODO change to production after code review
 22     BASE = 'LOCAL'  # options are LOCAL, STAGING, STAGING2, or PRODUCTION
 23     
 24     
(Pdb) POPULAR_LINKS_NODE
'8w6qe'
(Pdb) NEW_AND_NOTEWORTHY_LINKS_NODE
'su3q5'
(Pdb)
```